visitor/api.py

```diff
@@ -9,15 +9,15 @@ class VisitorList(ListCreateAPIView):
 
     queryset = Visitor.objects.all()
     serializer_class = VisitorSerializer
-    authentication_classes = (BasicAuthentication, )
```

Comment:
Why the following files are committed ??  What are the changes done ? Commit message for the same ? If any changes are there?